imports/plugins/included/connectors-shopify/server/hooks/orders.js

```diff
@@ -0,0 +1,278 @@
+/* eslint camelcase: 0 */
+import accounting from "accounting-js";
+import Shopify from "shopify-api-node";
+import { Reaction, Logger } from "/server/api";
+import { Orders, Shops } from "/lib/collections";
+import { getApiInfo } from "../methods/api";
+
+/**
+ * @private
+ * @summary Mark orders as exported after export
+ * @param {Object} exportedOrder the converted order
+ * @param {String} shopId - the shopId to attach
+ * @param {Object} order - the order to be marked
+ */
+function markExported(exportedOrder, shopId, order) {
+  Orders.update({ _id: order._id }, {
+    $push: {
+      exportHistory: {
+        status: "success",
+        dateAttempted: new Date(),
+        exportMethod: "reaction-connectors-shopify",
+        destinationIdentifier: exportedOrder.id,
+        shopId: shopId
+      }
+    }
+  });
+}
+
+/**
+ * @private
+ * @summary Mark orders as exported after export
+ * @param {Object} doc - the order to mark as failed
+ */
+function markExportFailed(doc) {
+  const order = Orders.findOne(doc._id); // only this object has the original transforms defined
+  const shops = Object.keys(order.getItemsByShop());
+  shops.forEach((shopId) => {
+    Orders.update({ _id: order._id }, {
+      $push: {
+        exportHistory: {
+          status: "failed",
+          dateAttempted: new Date(),
+          exportMethod: "reaction-connectors-shopify",
+          shopId: shopId
+        }
+      }
+    });
+  });
+}
+
+/**
+ * @private
+ * @summary build a new object for export to Shopify
+ * @param {Object} doc - the order to convert
+ * @param {Number} index - the sequence number of billing/shipping records we are processing
+ * @param {String} shopId - the Id of the shop we are processing for
+ * @returns {Object} shopifyOrder - the converted order
+ */
+function convertRcOrderToShopifyOrder(doc, index, shopId) {
```

Comment:
I'd prefer using `convertReactionOrderToShopifyOrder` unless that's so long that it's breaking something